src/components/atoms/button.tsx

```diff
@@ -61,6 +61,10 @@ export const Button: React.FC<ButtonProps> = ({
     }
   }
 
+  if (type === 'default') {
```

Comment:
Why don't you just add it to the default button shadow (as a color directly) ? Further more if the button is disabled are you adding this on top of +66?
Didn't look into how it should be done, just asking question here :P

Reply:
good points, thanks! I added it as color